### ISSUE TITLE
Modify CSS of generated pages

### ISSUE THREAD

**Issue by david-a-wheeler** [ossf/wg-best-practices-os-developers#160]
We need to modify the CSS for the generated pages under /docs to "look like OpenSSF".

This will probably involve modifying assets/css/style.scss as well as some other pages.

Eventually we should be able able to import a common CSS so it could be reused by different sites.

**Comment by jennbly** [comment]
Great idea! Yes, first draft of the OpenSSF brand guidelines with the OpenSSF logo/colors/fonts are available here: https://openssf.org/brand-guidelines/

**Comment by hythloda** [comment]
I submitted a request with the web design team for the CSS. I spent a while trying to pull it from the Wordpress site to no avail. Will share when they get back to me.

**Comment by ctcpip** [comment]
is there a good place to bring up a11y issues with the brand guidelines?  both the fonts are problematic for accessibility:

**Cairo:**
![Screenshot 2023-06-07 at 2 55 15 PM](https://github.com/ossf/wg-best-practices-os-developers/assets/15326526/18ee9aed-c6f4-4cc9-8b7e-348037ed419a)

**Teko:**
![Screenshot 2023-06-07 at 2 55 08 PM](https://github.com/ossf/wg-best-practices-os-developers/assets/15326526/d9d4ad79-e4cd-4c8b-881c-e920f458435f)

**Comment by jennbly** [comment]
Could you elaborate further on what the concern is? This is something we can bring up with our design and web teams to investigate.

**Comment by ctcpip** [comment]
@jennbly the legibility and readability of those fonts is not great -- the screenshots there are meant to demonstrate that.  using a better font would make OpenSSF web content more friendly for people with visual and cognitive disabilities.

**Comment by ctcpip** [comment]
I'll note that the OpenSSF website uses a different font (Poppins) -- which is more legible and readable but still leaves much to be desired:

![Screenshot 2023-06-07 at 5 17 25 PM](https://github.com/ossf/wg-best-practices-os-developers/assets/15326526/ddc71900-8bda-4f70-a8a6-c3abe0925575)

although the `18px` font size on the website helps a lot

**Comment by jennbly** [comment]
Agreed accessibility is very important. Thank you for the additional info. We will have a discussion with our design and web teams to determine the next steps forward.

**Comment by ctcpip** [comment]
As more guides are being published, I want to bring attention to the font a11y (and inconsistency) issue again.  Right now, guides appear with the Teko font for headers and Cairo font for body.  Cairo is ok, but the Teko font in particular has poor legibility.

That being said, the OpenSSF website uses the Poppins font for both headers and body.  Switching to this font for the guides would be an improvement and would bring them into consistent branding with the OpenSSF website as well as improving legibility and readability.  A win-win!

I suggest that this is an easy change we can make now.  Further discussion on the best fonts, as well as any other a11y issues, can take place in the near future.

**Comment by hythloda** [comment]
https://www.figma.com/file/AHpZECBYgXd3toYc1jC7lE/OpenSSF-Brand-Guidelines?type=design&node-id=427-5024&mode=design

Which one do we like!?

**Comment by hythloda** [comment]
1: <img width="621" alt="Screenshot 2023-08-10 at 1 36 16 PM" src="https://github.com/ossf/wg-best-practices-os-developers/assets/51727488/839a2871-500a-45b2-b25a-b6fe7e8d8a8e">

2: <img width="595" alt="Screenshot 2023-08-10 at 1 36 27 PM" src="https://github.com/ossf/wg-best-practices-os-developers/assets/51727488/c8420fe3-d509-4384-a276-3a2886850ddc">

3: 
<img width="601" alt="Screenshot 2023-08-10 at 1 36 53 PM" src="https://github.com/ossf/wg-best-practices-os-developers/assets/51727488/b89f16c4-74bf-4e81-bf30-719b9d06015b">

4:
<img width="612" alt="Screenshot 2023-08-10 at 1 37 02 PM" src="https://github.com/ossf/wg-best-practices-os-developers/assets/51727488/d7569c0e-4bd9-4e54-9a95-d4a96efc9526">

**Comment by ctcpip** [comment]
IBM Plex and Cairo by far are preferable.  Especially when looking at the headers in places like `Events`, `Training`, `Working Groups`, and `We're Hiring!`, as well as the bottom paragraph that starts with `Open source software is pervasive...`

**Comment by hythloda** [comment]
I like 1 and 3 the best as well, with a slight preference towards 3 because of the `Events`, `Training`, `Working` sections from the link. I just put the image encase the link didn't work for some people.

**Comment by jennbly** [comment]
In terms of readability - I have a strong preference for 3 that will also give us two fonts (both IBM PLex and Cairo) to work with in all of our materials.

**Comment by david-a-wheeler** [comment]
I have a minor preference for 3, it seems the clearest.

**Comment by Cheukting** [comment]
I am no designer, but I agree with the support for a11y. I guess it is 3 then? (both 1 and 3 look nice to my eyes, but I cannot tell much difference)

**Comment by david-a-wheeler** [comment]
@jennbly - please let us know when there's a final decision!

**Comment by david-a-wheeler** [comment]
Full modification blocked until we know if we're moving off WordPress.

**Comment by SecurityCRob** [comment]
@david-a-wheeler any update here?

**Comment by david-a-wheeler** [comment]
Not at this time. I've been waiting to hear more from Bennett.

**Comment by david-a-wheeler** [comment]
It has been decided that we'll stay on WordPress for the openssf.org site for the foreseeable future. There were some security concerns about how it had been deployed a while back, but I understand that work has gone into hardening its deployment. While WordPress is often mentioned in vulnerability reports, the core software has undergone a lot of review; the primary problem with WordPress is that many third-party plug-ins have *terrible* security. The solution is to review before adding a plug-in & don't just load random bits... which is the same solution for any other software :-).

**Comment by david-a-wheeler** [comment]
The generated pages for https://best.openssf.org use the OpenSSF fonts, so that's done.

We don't normally use colors, so I *think* the only thing currently left to do to [set link colors](https://www.w3schools.com/css/css_link.asp) in file `docs/assets/css/styles.css`. E.g., this format:

~~~~
/* unvisited link */
a:link {
  color: red;
}

/* visited link */
a:visited {
  color: green;
}

/* mouse over link */
a:hover {
  color: hotpink;
}

/* selected link */
a:active {
  color: blue;
}
~~~~

But instead of "green" and "blue" use the colors per https://openssf.org/about/brand-guidelines/

Anyone have preferences for which colors should apply to which? You want dark colors (like purple) for contrast with the white background.

**Comment by david-a-wheeler** [comment]
The [OpenSSF dark colors](https://openssf.org/about/brand-guidelines/) are:

* Prince Purple #604693
* Purple #45208C
* Royal Purple #2A146B
* Deep Purple #170D34
* Marine #007FA7
* Navy #04335F
* Dark Teal #00777E

I suggest selecting the OpenSSF colors closest to the existing colors. It might be wise to have 4 different colors for the 4 different link cases, to maximize information provided to users through color (if they can see color).
